File: android/src/main/kotlin/io/openim/flutter_openim_sdk_ffi/models/OfflinePushInfo.kt
package io.openim.flutter_openim_sdk_ffi.models

class OfflinePushInfo {
    var title: String? = null
    var desc: String? = null
    var ex: String? = null
    private var iOSPushSound: String? = null
    private var iOSBadgeCount = false
    fun getiOSPushSound(): String? {
        return iOSPushSound
    }

    fun setiOSPushSound(iOSPushSound: String?) {
        this.iOSPushSound = iOSPushSound
    }

    fun isiOSBadgeCount(): Boolean {
        return iOSBadgeCount
    }

    fun setiOSBadgeCount(iOSBadgeCount: Boolean) {
        this.iOSBadgeCount = iOSBadgeCount
    }
}

File: ios/Classes/models/OfflinePushInfo.swift
public class OfflinePushInfo {
    var title: String?
    var desc: String?
    var ex: String?
    private var iOSPushSound: String?
    private var iOSBadgeCount = false
    
    func getiOSPushSound() -> String? {
        return iOSPushSound
    }
    
    func setiOSPushSound(_ iOSPushSound: String?) {
        self.iOSPushSound = iOSPushSound
    }
    
    func isiOSBadgeCount() -> Bool {
        return iOSBadgeCount
    }
    
    func setiOSBadgeCount(_ iOSBadgeCount: Bool) {
        self.iOSBadgeCount = iOSBadgeCount
    }
}
